pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -166,8 +167,11 @@ func (pl *InterPodAffinity) getExistingAntiAffinityCounts(ctx context.Context, p
 		if len(topoMap) != 0 {
 			topoMaps[atomic.AddInt32(&index, 1)] = topoMap
 		}
+		return nil
+	}
+	if err := pl.parallelizer.Until(ctx, len(nodes), processNode, pl.Name()); err != nil {
+		logger.Error(err, "get an error during getExistingAntiAffinityCounts")
```

Comment:
```suggestion
		logger.Error(err, "Failed processing nodes in getExistingAntiAffinityCounts")
```

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message

Reply:
But it also wouldn't bother me to do:

```golang
_ = pl.parallelizer.Until(ctx, len(nodes), processNode, pl.Name())
```

Since it can be easily seen that this function doesn't return any errors.

Reply:
> But it also wouldn't bother me to do:

Make sense. Initially @sanposhiho commented that errors should be handled even though it doesn't return any errors. That's errors are handled here.